data/successors/successor weapons.txt

```diff
@@ -0,0 +1,343 @@
+	"mass" 22
+	"outfit space" -22
+	"weapon capacity" -22
+	"gun ports" -1
+	weapon
+		sprite "projectile/bimodal"
+			"frame rate" 5
+			"start frame" 0
+			"no repeat"
+		sound "bimodal"
+		"hit effect" "seeker impact"
+		"hit effect" "electron impact" 3
+		"fire effect" "moonbeam fleck" 4
+		"inaccuracy" 1
+		"velocity" 30
+		"lifetime" 15
+		"range override" 900
+		"burst count" 12
+		"burst reload" 6
+		"reload" 30
+		"firing energy" 80
+		"firing heat" 120
+		# Because parent projectiles inherit the damage of their submunitions, we must subtract the total damage of all submunitions from the parent projectile to get the desired behavior (different damage profiles over different ranges).
+		"shield damage" -94
+		# 50 (parent) - 3*48 (child)
+		# close: 50 shield damage
+		# far: 144 shield damage
+		"hull damage" 75
+		# 135 (parent) - 3*20 (child)
+		# close: 135 hull damage
+		# far: 60 hull damage
+		"hit force" 120
+		"cluster"
+		"submunition" "Bimodal Switch" 3
+	description `Precise and deadly, the "Ojet" Bimodal Coilgun has been the High Houses' armament of choice since long before humans ever took to the stars. It was designed quite simply as a perfect weapon; it is said that by the time the empire of the Predecessors fell, they had seen no need to install any other weapon on their ships for over a thousand years.`
+	description `The "Ojet" fires hyperdense packets of charged particles that disperse as they travel: up close the concentrated bolt pierces armor, while at a distance the ionic spray destabilizes shielding. Together, a ship equipped with these weapons can adjust its damage profile just by changing its engagement range, allowing it to be highly effective against any foe.`
+
+outfit `Bimodal Coilgun Turret`
+	category "Turrets"
+	cost 2950000
+	thumbnail "outfit/bimodal coilgun turret"
+	licenses
+		"High Houses"
+	"mass" 55
+	"outfit space" -55
+	"weapon capacity" -55
+	"turret mounts" -1
+	weapon
+		"hardpoint sprite" "hardpoint/bimodal turret"
+		"hardpoint offset" 9.
+		sprite "projectile/bimodal"
+			"frame rate" 5
+			"start frame" 0
+			"no repeat"
+		sound "bimodal"
+		"hit effect" "seeker impact"
+		"hit effect" "electron impact" 3
+		"fire effect" "moonbeam fleck" 4
+		"inaccuracy" 1
+		"velocity" 30
+		"lifetime" 15
+		"range override" 900
+		"turret turn" 3.5
+		"burst count" 24
+		"burst reload" 3
+		"reload" 15
+		"firing energy" 80
+		"firing heat" 120
+		"firing force" 25
+		"shield damage" -94
+		"hull damage" 75
+		"hit force" 120
+		"cluster"
+		"submunition" "Bimodal Switch" 3
+	description `Successor warships have traditionally relied on fixed weaponry to deal with their foes, but the High Houses do produce a turreted platform for the "Ojet" Coilgun so that their larger and slower vessels can maintain their combat effectiveness against more manuverable targets. It mounts two of the weapons side by side, rigged to fire double the amount of shots per burst to keep them in sync with the fixed hardpoints.`
```

Comment:
```suggestion
	description `Successor warships have traditionally relied on fixed weaponry to deal with their foes, but the High Houses do produce a turreted platform for the "Ojet" Coilgun so that their larger and slower vessels can maintain their combat effectiveness against more maneuverable targets. It mounts two of the weapons side by side, rigged to fire double the amount of shots per burst to keep them in sync with the fixed hardpoints.`
```